src/resources/MachineLearning/SmartSnippetsConfiguration/SmartSnippetsConfigurationInterfaces.ts

```diff
@@ -33,7 +33,7 @@ export interface SmartSnippetsConfigurationModel {
     documentTypes?: DocumentType[];
 }
 
-export type DocumentRequirementStatus = 'OK' | 'INSUFFICIENT_DOCUMENTS';
+export type SmartSnippetsDocumentRequirementStatus = 'OK' | 'INSUFFICIENT_DOCUMENTS';
```

Comment:
The other breaking change is changing the value of `baseUrl`. in[ a similar PR ](https://github.com/coveo/platform-client/pull/401) I made for Smart Snippets I was advised to make the breaking change so I did it here too. It would've been better to bundle them into one major bump but that's not really how this repo works.
So if I'm making one breaking change already I figured might as well do this one too.

Reply:
Why declare 2 identical types with 2 different variable names?

Reply:
Because other than their similarity, there's nothing about what they represent that would indicate they'll always remain the same. 
The requirements for a valid case classification document group might change and introduce a new status while the requirements for a smart snippets document group might not. then we'd be back here again.

Also the type is exported from platform-client so if a user wants to import the status type and sees `DocumentRequirementStatus` and `CaseClassificationDocumentRequirementStatus`, it is still unclear which one is correct. 

It's my mistake, I should've had the foresight to name it correctly the first time. So this is what I propose to fix it.

Reply:
thank for explanations ;)

Reply:
Ok, let's rename the type then.